hapic/context.py

```diff
@@ -106,8 +107,101 @@ def serve_directory(
         """
         raise NotImplementedError()
 
+    def handle_exception(
+        self,
+        exception_class: typing.Type[Exception],
+        http_code: int,
+    ) -> None:
+        raise NotImplementedError()
```

Comment:
docstring should be described in the interface, otherwise you do not know what should be implemented

Reply:
done